src/map/skill.c

```diff
@@ -94,7 +95,8 @@ int skill_splash_target(struct block_list* bl)
 }
 
 /// Returns the id of the skill, or 0 if not found.
-int skill_name2id(const char* name) {
+int skill_name2id(const char* name)
```

Comment:
It's indeed against our Coding style, and it should be char *name, so you're welcome to pull request such changes when you find!